### PR TITLE
Use helmify to generate the helm chart

### PR DESCRIPTION
**Pull request by zalsader**
Use [helmify](https://github.com/arttor/helmify) to generate the helm chart. The tool is compatible with kubebuilder.

Resolves: KSD-152

### CODE REVIEW

**Comment by zalsader** [comment]
This is built on top of  #9 because there are features I need from there. Once that merges, the base should be changed to main.

**Review by galexrt — APPROVED** [review]

**Comment by zalsader** [comment]
* adds a do not merge label *
* accidentally meges the PR * 🤦🏽 

Since I can't reopen a PR, please review this one: #11

**Pull request merged**
